VideoLocker/src/test/java/org/edx/mobile/test/http/HttpBaseTestCase.java

```diff
@@ -52,6 +55,8 @@
     //there are third party extension to handle conditionally skip some test programmatically.
     //but i think it is not a good idea to introduce more libs only for this purpose.
     protected boolean shouldSkipTest = false;
+    @Mock
+    protected UserPrefs userPrefs;
```

Comment:
Why does this need to be mocked? You could just inject the actual `UserPrefs` singleton where you like.